src/num.py

```diff
@@ -40,17 +41,36 @@ def add(self,v):
                 self.has[pos]=float(v)
             
     def div(self):
+        def __percentile(lst,n):
+            size = len(lst)
+            return sorted(lst)[int(math.ceil((size * n) / 100)) - 1]
+        
         a = self.nums();
         total = sum(a)
         length = len(a)
         mean = total / length
         std = (sum([((x - mean) ** 2) for x in a]) / length) ** 0.5
-        return std
+        std1 = (__percentile(a,90) - __percentile(a,10))/2.58
+        return std1
     
     def mid(self):
         lst = self.nums()
         if len(lst)%2==0:
             mid = (lst[len(lst)//2]+lst[(len(lst)//2)-1])/2
         else:
             mid = lst[len(lst)//2]
-        return mid 
\ No newline at end of file
+        return mid 
+
+if __name__ == "__main__":
```

Comment:
Remove this. Main function only for local testing

Reply:
Updated